SocialFilter/sections/specific.txt

```diff
@@ -36,6 +36,7 @@ sonsaat.com.tr,okudunmuhaberi.com,spooor.com,mavididim.com.tr,filmfiz.net,filmw.
 kevinpurcell.org,015.by,airkule.com,altyazi.net,anilibria.tv,asyaspor21.tv,bartinolay.com,bedava-film.com,belaruspartisan.by,belgeselciyiz.com,belsat.eu,brauer-bund.de,btcnewz.com,businessturkeytoday.com,cj3z.com,corgit.xyz,creatorlabs.net,cwbaltimore.com,daemon-hentai.com,der-postillon.com,desimartini.com,discoverskills.com,discoverychannel.com.tr,dizilara.org,dizitube.net,dizivay.com,efelerhaber.com,egeyonhaber.com,elazigyenihaber.com,ellegirl.ru,erkekadam.org,erzurumajans.com,eurolab.ua,filmakinesi.net,filmold.com,fontanka.ru,forums.flyer.co.uk,fox38corpuschristi.com,fox47.com,foxkansas.com,foxrichmond.com,gazetaby.com,gercekizmir.com,gidavitrini.com.tr,halklailiskiler.com.tr,hizlifilmizlehd.com,inforesist.org,jetvis.ru,karsolay.com,kbsi23.com,kdsm17.com,kg-portal.ru,kinopoisk.userecho.com,kp.ua,losst.ru,magazinekibi.com,manisamanset.gen.tr,mforos.com,mogilev.online,muchohentai.com,multifilemirror.com,mvs.gov.ua,mywebtimes.com,news-journalonline.com,nk.org.ua,noticierodigital.com,ntp-servers.net,obltv.ru,orbita.dn.ua,pctip.ru,podrobnosti.ua,ppt.ru,proglang.su,queenbet4.tv,raleighcw.com,real-vin.com,rmj.ru,rusvesna.su,sanatatak.com,seirsanduk.us,simdifilmizle.com,sonkaleizmir.com,sp-mobile-blog.com,star64.tv,tomiks33.ru,torrentbe.net,torrentrg.org,tqm.com.ua,tribundergi.com,tva.in.ua,utofiles.com,wcti12.com,womanel.com.ua,wpgh53.com,wtto21.com,wutv29.com,xn----8sbcm1bmcar0m.xn--p1ai,earthsky.org,hentairules.net,actualcomment.ru,nkj.ru,tayga.info,hornygamer.com,blog.jetbrains.com##.twitter-timeline
 !
 focus.ua##.c-post__social
+ilovepdf.com##.card
```

Comment:
`.card` isn't as unique as `.download__add`, so it's better to replace it to `ilovepdf.com##.download__add`.

Reply:
Nvm, I made changes.

Reply:
@zloyden I checked again, and for me the `.download__add` rule does not work, but the `.card` rule works.

Reply:
It works fine on my end. Can you please check `ilovepdf.com#?#.container > div[class]:has(> div.card div.share)` in the User rules?

Reply:
The rule `ilovepdf.com#?#.container > div[class]:has(> div.card div.share)` works for me.

Reply:
Fine, it was added recently. Filters update should be available in 1-2 hour.

Reply:
Why does the `ilovepdf.com##.download__add` rule work only in the extension, but not in the application? Is this an app bug or platform limitation?

Reply:
The extension and the app have a different approach to blocking elements in such cases, but we have a possibilites to block it via app. Thanks for your feedback:)